proxy.js

```diff
@@ -1024,6 +1024,10 @@ function activateHTTP() {
 				for (let minerID in activeWorkers[workerID]){
                 			if (!activeWorkers[workerID].hasOwnProperty(minerID)) continue;
 					let miner = activeWorkers[workerID][minerID];
+					if (!miner) continue;
```

Comment:
Is it really can happen? I mean not defined miner.

Reply:
Thanks for clarification. Now I understand why if (!miner.identifier) can not avoid error. (I add one more if !miner continue after try and fail).
Do you think it causes error because we run http on multiple threads ?

Reply:
I'm noob. Author or not isn't important. I just want your repo can grow. More and more people come and contribute in addition to your work.